### PR TITLE
Pass seed in to profile contents and channels

### PR DESCRIPTION
**Pull request by broskoski**
Closes part of #1544 (random sort should be different on refresh)

### CODE REVIEW

File: react/components/ProfileContents/index.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ export default class ProfileContents extends Component {
     per: 12,
     hasMore: true,
     q: null,
+    seed: Date.now(),
```
Comment (dzucconi) [inline react/components/ProfileContents/index.js]: Won't the seed be different between ssr and the client-side paging?

Comment (broskoski) [reply react/components/ProfileContents/index.js]: Ah yeah, good point. Hm, any ideas?

Comment (dzucconi) [reply react/components/ProfileContents/index.js]: Set it as a cookie or push it into the query params so that it's at a routing level.

**Comment by broskoski** [comment]
Closing this PR and starting fresh

**Pull request closed without merging**